doc/man3/EVP_CIPHER_meth_new.pod

```diff
@@ -204,17 +204,32 @@ EVP_CIPHER_CTX_get_cipher_data().
 This cleanup function is called by EVP_CIPHER_CTX_reset() and
 EVP_CIPHER_CTX_free().
 
+EVP_CIPHER_meth_set_set_asn1_params() sets the function for B<cipher>
+to set the AlgorithmIdentifier "parameter" based on the passed cipher.
+This function is called by EVP_CIPHER_param_to_asn1().
+EVP_CIPHER_meth_set_get_asn1_params() sets the function for B<cipher>
+that sets the cipher parameters based on an ASN.1 AlgorithmIdentifier
+"parameter".
+Both these functions are needed when there's a need for custom data
```

Comment:
"there's" change to "there is "

Reply:
Okie, will do.  I'll have you note, though, that we have no agreement on this.  Some of us don't want contractions, others aren't bothered by them.